src/Diagnostics.RuntimeHost/Program.cs

```diff
@@ -64,7 +63,8 @@ public static IHostBuilder CreateHostBuilder(string[] args)
                             .AddAzureKeyVault(
                                 keyVaultUri,
                                 keyVaultClient,
-                                new DefaultKeyVaultSecretManager());
+                                new DefaultKeyVaultSecretManager())
+                            .AddAndInstallAzureKeyVaultCertificates(keyVaultUri, keyVaultClient);
```

Comment:
Just so I understand, in production, the certs will be synced with Prod keyvault, correct?

Reply:
Nope, the certs would have to be installed on the web app (App Service). Antares takes care of setting up the cert store for us and removes the KV dependency. Before the website starts, cert store is already set up and the app should have all that it needs to contact AAD. 

This piece is us mimicking Antares capability of syncing certificates for our local dev to work. 

I have already manually installed the certs required for Kusto, ASC and Http data provider on each of our runtimehost web apps.